CMake/Utils/AndroidMacros.cmake

```diff
@@ -8,6 +8,26 @@
 #-------------------------------------------------------------------
 
 
+if(OGRE_BUILD_ANDROID_JNI_LIBS_CMAKE)
+    if(NOT TARGET cpufeatures)
+        add_library(cpufeatures STATIC ${ANDROID_NDK}/sources/android/cpufeatures/cpu-features.c)
+        target_link_libraries(cpufeatures dl)
+    endif(NOT TARGET cpufeatures)
+
+    if(NOT TARGET native_app_glue)
```

Comment:
the condition when the native_app_glue is built is hard to follow, please rather copy the according parts from here:
https://github.com/taka-no-me/android-cmake/blob/master/AndroidNdkModules.cmake